src/lightmap.cpp

```diff
@@ -492,6 +492,46 @@ bool map::pl_line_of_sight( const tripoint &t, const int max_range ) const
 
 #include "messages.h"
 
+typedef ShadowcastingSlope slope;
+
+slope::ShadowcastingSlope( int x, int y )
```

Comment:
Why are you even doing a typecast? If the name is too long make it not so
long, otherwise just use it as-is.